contrib/google.golang.org/grpc/appsec_test.go

```diff
@@ -389,7 +390,7 @@ func (s *appsecFixtureServer) Ping(ctx context.Context, in *FixtureRequest) (*Fi
 	md, _ := metadata.FromIncomingContext(ctx)
 	ids := md.Get("user-id")
 	if err := pappsec.SetUser(ctx, ids[0]); err != nil {
-		if e, ok := err.(pappsec.MonitoringError); ok {
+		if e, ok := err.(*grpcsec.MonitoringError); ok {
 			err = status.Error(codes.Code(e.GRPCStatus()), e.Error())
 		}
```

Comment:
Same as above, blocking must be automated in our middleware:
```suggestion
```

Reply:
Fixed in [905cd39](https://github.com/DataDog/dd-trace-go/pull/2044/commits/905cd396e9fdc970f2e125216c2ce8479450939a)